pkg/types/apis/management.cattle.io/v3/tools_system_images.go

```diff
@@ -17,7 +17,7 @@ var (
 			JenkinsJnlp:   m("jenkins/jnlp-slave:3.35-4"),
 			AlpineGit:     m("rancher/pipeline-tools:v0.1.14"),
 			PluginsDocker: m("plugins/docker:18.09"),
-			Minio:         m("minio/minio:RELEASE.2019-09-25T18-25-51Z"),
+			Minio:         m("minio/minio:RELEASE.2020-07-02T00-15-09Z"),
```

Comment:
I see few more recent tags/releases under https://github.com/minio/minio/tags

Dont we want to update to the latest?